CHANGELOG.md

```diff
@@ -6,10 +6,62 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-### Changed
+## [1.2.0-rc.2] - 2024-06-11
+
+> TRUE or FALSE, it's a problem!
 
+### Important Notes
  * libcontainer/cgroups users who want to manage cgroup devices need to explicitly
    import libcontainer/cgroups/devices. (#3452, #4248)
+ * If building with Go 1.22.x, make sure to use 1.22.4 or a later version.
+   (see #4233 for more details)
+
+### Added
+
+ * CI: add actuated-arm64. (#4142)
+
+### Fixed
+
+ * Fix tests in debian testing (mount_sshfs.bats) - minimal fix. (#4245)
+ * Set the default value of CpuBurst to nil instead of 0. (#4211)
+ * Fix set nofile rlimit error. (#4265, #4282, #4290)
```

Comment:
I'd say "runc exec: fix setting rlimit_nofile" as the fix is specific for runc exec.